app/views/shared/_citations.html.erb

```diff
@@ -21,12 +21,10 @@
   <% end %>
   <meta name="citation_publication_date" content="<%= @presenter.date_created.first %>" />
   <meta name="citation_pdf_url" content="<%= @presenter.download_url %>" />
-  <!-- Hyrax does not yet support these metadata -->
-  <!--
-    <meta name="citation_journal_title" content=""/>
-    <meta name="citation_volume" content=""/>
-    <meta name="citation_issue" content=""/>
-    <meta name="citation_firstpage" content=""/>
-    <meta name="citation_lastpage" content=""/>
-  -->
+  <meta name="citation_journal_title" content="<%= @presenter.citation_journal_title if @presenter.respond_to?(:citation_journal_title) %>" /> 
+  <meta name="citation_volume" content="<%= @presenter.citation_volume if @presenter.respond_to?(:citation_volume) %>" /> 
+  <meta name="citation_issue" content="<%= @presenter.citation_issue if @presenter.respond_to?(:citation_issue) %>" /> 
+  <meta name="citation_firstpage" content="<%= @presenter.citation_firstpage if @presenter.respond_to?(:citation_firstpage) %>" /> 
+  <meta name="citation_lastpage" content="<%= @presenter.citation_lastpage if @presenter.respond_to?(:citation_lastpage) %>" /> 
+  <meta name="citation_dissertation_institution" content="<%= "Lafayette College" if @presenter.respond_to?(:advisor) %>" />

```

Comment:
could you use `if @presenter.model_name == 'StudentWork'` instead of checking :advisor? just in case that field gets added to another work type in the future.